src/components/TutorialMetadata.tsx

```diff
@@ -26,14 +25,14 @@ export enum Skill {
 export const getSkillTranslationId = (skill: Skill): TranslationKey =>
   `page-tutorial-${Skill[skill.toUpperCase() as keyof typeof Skill]}`
 
-const TutorialMetadata: React.FC<IProps> = ({ tutorial }) => {
-  const { language } = useI18next()
 
-  const frontmatter = tutorial.frontmatter
+const TutorialMetadata: React.FC<IProps> = ({ frontmatter, timeToRead }) => {
```

Comment:
Nice to have: new ts conventions
```suggestion
const TutorialMetadata = ({ frontmatter, timeToRead }: TutorialMetadataProps) => {
```